app/scripts/migrations/154.ts

```diff
@@ -0,0 +1,57 @@
+import { AVAILABLE_MULTICHAIN_NETWORK_CONFIGURATIONS } from '@metamask/multichain-network-controller';
+import { BtcScope, SolScope } from '@metamask/keyring-api';
```

Comment:
these values need to be copied/inlined into this migration file

Reply:
https://github.com/MetaMask/metamask-extension/pull/31702/commits/8890d7440ee7a7460d36b9a3b0ee6d8f6a2b51cf